ui/app/templates/components/dashboard/quick-actions-card.hbs

```diff
@@ -38,13 +39,15 @@
         class="is-flex-grow-1"
         @selectLimit="1"
         @models={{array this.searchSelectParams.model}}
-        @backend={{this.selectedEngine.id}}
         @placeholder={{this.searchSelectParams.placeholder}}
         @disallowNewItems={{true}}
         @onChange={{this.handleActionSelect}}
         @fallbackComponent="input-search"
-        @nameKey={{this.searchSelectParams.nameKey}}
         @disabled={{not this.searchSelectParams.model}}
+        @nameKey={{this.searchSelectParams.nameKey}}
```

Comment:
I think we should pass `@shouldRenderName={{true}}` so that these name keys render